crates/sui-graphql-rpc/src/types/query.rs

```diff
@@ -141,6 +144,7 @@ impl Query {
             .extend()
     }
 
+    /// The checkpoints that exist in the network.
```

Comment:
also not sure about these comments. Do we want the comment to mention why we'd use checkpoints over checkpoint (multi-query, can paginate through, etc.) LIke, "Fetch and paginate through information on multiple checkpoints" since the language for the singleton query is, " Fetch checkpoint information by sequence number or digest (defaults to the latest available
checkpoint).
"""
checkpoint(id: CheckpointId): Checkpoint"

Reply:
Great point, thanks!